makeflow/src/makeflow.c

```diff
@@ -1652,9 +1690,12 @@ int main(int argc, char *argv[])
 
 	if(clean_mode) {
 		printf("cleaning filesystem...\n");
+
 		makeflow_clean(d);
+        makeflow_remove_sandbox(logfilename);
 		unlink(logfilename);
 		unlink(batchlogfilename);
+               
```

Comment:
Please avoid checking in spurious whitespace changes, since they make it harder to figure out the history of the code.